lib/jobs/message_latency_report.ex

```diff
@@ -123,15 +135,39 @@ defmodule Jobs.MessageLatencyReport do
         station: station
       ]
     end)
-    |> Enum.sort_by(fn row -> row[:seconds] end)
-    |> Enum.with_index()
```

Comment:
We don't need to sort + index the rows until after we've filtered out the irrelevant dates so I moved this into `get_stats/2`